model/src/main/java/de/danoeh/antennapod/model/feed/VolumeAdaptionSetting.java

```diff
@@ -32,4 +34,18 @@ public int toInteger() {
     public float getAdaptionFactor() {
         return adaptionFactor;
     }
+
+    public static boolean BOOST_SUPPORTED = false;
+
+    static  {
+        final AudioEffect.Descriptor[] audioEffects = AudioEffect.queryEffects();
+        if (audioEffects != null) {
+            for (AudioEffect.Descriptor effect : audioEffects) {
+                if (effect.type.equals(AudioEffect.EFFECT_TYPE_LOUDNESS_ENHANCER)) {
+                    BOOST_SUPPORTED = true;
+                    break;
+                }
+            }
+        }
+    }
```

Comment:
This will make the app start slower. We already do so many initializations in static variables. I don't think that's a good thing because it prevents using the normal framework methods for loading screens (we use an ugly hack to get our loading screen). I would just make this a normal function, it shouldn't be performance critical to evaluate this multiple times when needed I guess.